lighthouse-core/report/html/report-styles.css

```diff
@@ -1105,6 +1123,16 @@
   --gauge-circle-size: var(--gauge-circle-size-big);
 }
 
+.lh-sticky-header--visible > .lh-gauge__wrapper--fail > .lh-gauge__svg-wrapper {
+  background-image: url("data:image/svg+xml,%3Csvg width='36' height='36' viewBox='0 0 36 36' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M17 26H20V23H17V26Z' fill='%23FF4E42'/%3E%3Cpath d='M17 20H20V11H17V20Z' fill='%23FF4E42'/%3E%3C/svg%3E");
```

Comment:
I don't think we need to modify the SVG here, just position it
```css
background-position: center;
background-repeat: no-repeat;
```

Reply:
What you mean by not modifying svg in this comment. Do you mean using same sized svg for both these cases,  and just scaling it up or down.
`.lh-sticky-header--visible > .lh-gauge__wrapper--fail > .lh-gauge__svg-wrapper`
&
`.lh-category-header .lh-gauge__wrapper--fail  .lh-gauge__svg-wrapper`

Reply:
I mean that we should reuse the same SVG data URL for all of these (kept in a CSS variable) and not mess with its `width`/`height`/`viewBox`.

In this case we don't even need to scale it up (though we will need to for the *large* gauge)